en/ha.rst

```diff
@@ -646,6 +647,12 @@ CUBRID checks hosts specified in **ha_ping_hosts** every hour; if there is a pro
 
 Configuring this parameter can prevent split-brain, a phenomenon in which two master nodes simultaneously exist as a result of the slave node erroneously detecting an abnormal termination of the master node due to unstable network status and then promoting itself as the new master.
 
+However, the "ping check" does not operate normally if the ICMP protocol is disabled. CUBRID provides **ha_tcp_ping_hosts** as an alternative to solve this problem.
+
+**ha_tcp_ping_hosts**
+
+**ha_tcp_ping_hosts** is a parameter that can be used as an alternative to **ha_ping_hosts** when the ICMP protocol is disabled. **ha_tcp_ping_hosts** operates like **ha_ping_hosts** except that the TCP layer is used instead of the IP layer for the "ping check". The default is **NULL**. A comma(,) is used to separate individual host names, and a colon(:) is used to separate a host name and a port number. So, the format of this parameter is like "ha_tcp_ping_hosts=host1:port1,host2:port2". In order to use the TCP ping normally, the TCP socket that can receive the requests should be opened in advance with the port number on the host set to **ha_tcp_ping_hosts** and the firewall should not block the requests. Additionally, if both **ha_tcp_ping_hosts** and **ha_ping_hosts** are set, **ha_tcp_ping_hosts** is ignored.
```

Comment:
```suggestion
**ha_tcp_ping_hosts** is a parameter that can be used as an alternative to **ha_ping_hosts** when the ICMP protocol is disabled. **ha_tcp_ping_hosts** works like **ha_ping_hosts** except that the TCP layer is used instead of the IP layer for the "ping check". The default is **NULL**. A comma(,) is used to separate individual host names, and a colon(:) is used to separate a host name and a port number. So, the format of this parameter is like "ha_tcp_ping_hosts=host1:port1,host2:port2". In order to use the TCP ping properly, a TCP socket that can receive the request must be opened in advance with the port number on the host specified to **ha_tcp_ping_hosts** and the firewall must not block the requests. The **ha_tcp_ping_hosts** is ignored if the **ha_ping_hosts** is also set.
```